sparksql-scalapb/src/main/scala/org/apache/spark/sql/catalyst/expressions/timestampExpressions.scala

```diff
@@ -0,0 +1,48 @@
+package org.apache.spark.sql.catalyst.expressions
```

Comment:
The two methods here look like they are just a special case of `Invoke` - are they really needed? I think they can be replaced with functions that return an instance of `Invoke`.